src/ctap/data_formats.rs

```diff
@@ -504,6 +504,22 @@ impl TryFrom<cbor::Value> for SignatureAlgorithm {
     }
 }
 
+/// The level of protection granted by the credProtect extension.
+///
+/// This level applies to resident credentials. The levels have the following
+/// implications:
+///
+/// UserVerificationOptional
+/// The credential is always discoverable, as if it had no protection level.
+///
+/// UserVerificationOptionalWithCredentialIdList
+/// The credential is discoverable with
+/// - an allowList,
+/// - an excludeList,
+/// - user verification.
+///
+/// UserVerificationRequired
+/// The credentials is discoverable with user verification only.
```

Comment:
Those documentation should be directly on the enum values.

```rust
/// Documentation of Foo
enum Foo {
    /// Documentation of Bar
    Bar,
    /// Documentation of Baz
    Baz,
}
```

Reply:
Done